src/components/NewMessage.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+    <div class="newmessage-block">
+      <div class="row input">
+        <form class="col s12 form">
+          <div class="row">
+            <div class="input-field col s12">
+              <textarea id="textarea2"
+                        class="materialize-textarea"
+                        data-length="120"
+                        maxlength="140"
+                        v-model="post.text"
+                        @input="clearErrors"
+              ></textarea>
+              <label for="textarea2" class="label">Text (max: 140)</label>
+            </div>
+          </div>
+          <div class="row">
+            <div class="input-field col s6">
+              <input id="input_text"
+                     type="text"
+                     data-length="10"
+                     v-model="post.hashtag"
+                     @input="clearErrors"
+              >
+              <label for="input_text">#</label>
+            </div>
+          </div>
+          <div class="error-block" :class="{'error-block--visible': errors.length}">
+            <p v-if="errors.length">
+              <b>{{ $t("errors.title") }}</b>
+            <ul>
+              <li v-for="error in errors" :key="error">{{ $t(error) }}</li>
+            </ul>
+            </p>
+          </div>
+          <div class="buttonPost">
+            <a class="waves-effect waves-light btn"
+               @click="checkForm"
+               :disabled="errors.length > 0"
+            >post</a>
+          </div>
+        </form>
+      </div>
+    </div>
+</template>
+
+<script>
+import axios from 'axios'
+export default {
+  name: 'NewMessage',
+  data () {
+    return {
+      errors: [],
+      post: {
+        text: '',
+        hashtag: ''
+      }
+    }
+  },
+  methods: {
+    checkForm (e) {
+      this.errors = []
+      if (!this.post.text) {
+        this.errors.push('Enter text')
+      }
+      if (!this.post.hashtag) {
+        this.errors.push('Enter hashtag plz')
+      } else if (!this.isHash(this.post.hashtag)) {
+        this.errors.push('Hashtag must be')
```

Comment:
no i18n?
`Hashtag must be` -> `Hashtag format should be #foo#bar`
Hashtag input may be empty. Only Text must be not empty